src/main/java/quotify_app/entities/CommonUser.java

```diff
@@ -0,0 +1,32 @@
+package quotify_app.entities;
+
+/**
+ * A simple implementation of the User interface.
+ */
+public class CommonUser implements User {
+
+    private final String name;
+    private final String password;
+    private final String email;
+
+    public CommonUser(String name, String password, String email) {
+        this.name = name;
+        this.password = password;
```

Comment:
No, this was just straight taken out lab-5, BUT tbh this is a good idea and we could build off this. Since CommonUser is already like the 'regular' user, we could have another implementation of the User interface that's like: GuestUser or something?

Reply:
I would recommend adding a comment about this somewhere in either the CommonUser class or the User class so whoever is assigned to creating the guestUser is aware.